github-actions/README.md

```diff
@@ -0,0 +1,87 @@
+# GitHub Actions
+
+## Benefits and options for Packit and users of Packit
+
+### Customization as a whole
+
+In case of providing Packit API via GitHub Actions it allows to run pipeline (SRPM → Copr →
+Testing Farm) after:
+
+- running some initial CI checks (saves resources on our side)
+- preparing archives (eliminates need for retries and failed jobs on our side, if they are blocked
+  by some external action)
+
+### SRPM builds
+
+- Allows user to **easily** customize the dependencies, i.e. does not require changes to be made to
+  the Sandcastle image.
+- Provides access to created SRPMs without dependency on Copr or Packit Service.
+
+## Disadvantages and caveats
+
+### SRPM builds
+
+- If we were to trigger Copr builds with SRPM created in GitHub Action we introduce blocking action of
+  acquiring the SRPM from GitHub.
+- No cache is present, which also introduces:
+  - slower build time
+  - potential time out, since Actions are time limited (iirc 24 hrs per action)
```

Comment:
this is a good point, since job scheduling would be completely out of our control